### PR TITLE
Reimplemented Probabilities Classifier in Tensorflow

### PR DESCRIPTION
**Pull request by anayebi**
Reimplemented the Tensorflow probabilities classifier in Tensorflow, which should allow for:
- a more flexible interface for adding more sophisticated decoding strategies
- can get slightly better results than the prior sklearn multinomial regression classifier.

### CODE REVIEW

**Comment by mschrimpf** [comment]
Closing due to inactivity. The classifier was however used with modifications in the [OST metric](https://github.com/brain-score/brain-score/blob/d03e50fec90a5a4a20b9919064df6babc1e11c40/brainscore/metrics/ost.py#L93).
